lib/iris/coord_systems.py

```diff
@@ -765,25 +809,21 @@ def __init__(
         )
 
         #: Altitude of satellite in metres.
-        # test if perspective_point_height may be cast to float for proj.4
         self.perspective_point_height = float(perspective_point_height)
+        # TODO: test if may be cast to float for proj.4
```

Comment:
NOTE: comment line can not appear after the '#:' attribute-docstring line : it prevents it from rendering.